working-w-copilot/copilot-in-projects-new-code.md

```diff
@@ -386,61 +344,65 @@ We could make the changes manually after moving the code to a file, but we can a
 
 <details>
   <summary>
-    We will leave marking properties as required as a personal exercise, but feel free to try out asking Copilot to help with the changes to the <code>Book</code> and <code>Author</code> <code>to_dict</code> functions. When you're done, expand this section to see the prompt we used and what Copilot suggested.
+    Feel free to try out asking Copilot to help with the changes to the <code>Book</code> and <code>Author</code> <code>to_dict</code> functions. When you're done, expand this section to see the prompt we used and what Copilot suggested.
   </summary>
 
   **Updated Prompt:**
-  > Please update the Book to_dict function to include the name of the author and a list of the names of the genres associated in the return dictionary, only if those pieces of data exist. Please update the Author to_dict function to include the title of each item in the list books.
+  > Please update the Book to_dict function to include only the name of the author and a list of the names of the genres associated in the return dictionary, if those pieces of data exist. Please update the Author to_dict function to include only the title of each item in the list books.
```

Comment:
Even though author and genre were absent from my copilot-generated to_dict, this prompt still got it to the same place.